README.md

```diff
@@ -22,15 +22,44 @@ This project was created with [`create-next-app`](https://github.com/vercel/next
 
 - nvm [Linux/macOS](https://github.com/nvm-sh/nvm), [Windows](https://github.com/coreybutler/nvm-windows)
 - [yarn](https://classic.yarnpkg.com/en/docs/install)
+- For Windows machines ensure the [VC_redist.x64](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) which is helpful with ensuring issues aren't encountered when trying to run project locally
```

Comment:
Can you explain what the Visual C++ Redistributable is for? Is it a prerequisite of one of the other tools? This is OK if not 🙂